app/_includes/md/enterprise/deploy-license.md

```diff
@@ -61,48 +70,75 @@ For more detail and options, see the
 {% endnavtab %}
 {% navtab Filesystem %}
 
-Securely copy the `license.json` file to your home directory on the filesystem
+You can provide a license file to Kong Gateway in any database-backed or DB-less
+deployment. This method cannot be used in hybrid mode.
```

Comment:
Can we make `This method cannot be used in hybrid mode.` _really_` stand out? I was reviewing this PR and missed this line, it was only when I remembered seeing a DOCU ticket that I specifically went looking for the info

Reply:
I am confused; why are we saying this can't be used in hybrid mode? If you create the file on your CP and on your DP it will get picked up by the Kong Gateway. The order of precedence the CP or DP will obtain the license is the following:

1. `KONG_LICENSE_DATA` environment variable.
2. `/etc/kong/license.json` file.
3. `KONG_LICENSE_PATH` environment variable which points to a file containing the license.
4. `/licenses` endpoint; recommended method of deployment.

Reply:
This is info from the ticket:

From the reporter:
> method-4: Directly deploy a license using the /licenses Admin API endpoint.
> License only can be distributed from CP to DP by using method-4

from your comment on the ticket: 
> the /licenses endpoint is the only method for pushing the license down to the data plane from the control plane through the payload messaging

Effectively, that means you have to use this method, otherwise you may end up with mismatched CP and DP nodes, no? One of the benefits of using a control plane/data plane setup is sending licenses to all DPs in the cluster, from what I've gathered. Do you see any reason that someone would _not_ do that?

Reply:
You can definitely end up with mismatched nodes if you don't distribute the license properly using methods 1 thru 3. I am just saying the verbiage "cannot" is misleading as you can use any of those 3 methods; is it just not automatic (e.g. handled by Kong Gateway) and more error prone if you use the other 3 methods for hybrid setups .